php/plain/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "deploio-examples/plain-php",
+    "description": "A simple example application to demonstrate deplo.io with PHP.",
+    "type": "project",
+    "require": {
+        "php": "^8.0",
+        "imagine/imagine": "^1.5",
+        "ext-gd": "*"
+    },
+    "license": "proprietary"
```

Comment:
This can be MIT or similar.

Reply:
i copied that from the symfony app, thinking you wanted it like this. i see that in the repository root, there is the apache license, which is also set in the github configuration. 

i changed it both for this as well as for the symfony app.